src/utils/item.ts

```diff
@@ -2,8 +2,6 @@
  * todo: use utils from a dedicated repo */
 import { UUID } from '@graasp/sdk';
 
-import CryptoJS from 'crypto-js';
-
 // eslint-disable-next-line no-useless-escape
 export const transformIdForPath = (id: UUID) => id.replace(/\-/g, '_');
```

Comment:
If you want you could even remove that to use from sdk.

Reply:
Yes will do !